langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -64,10 +79,21 @@ public isolated function isNaN(float x) returns boolean = @java:Method {
 
 # Returns the sum of zero or more float values.
 #
-# Result is NaN if any arg is NaN
+# ```ballerina
+# // Take the sum of multiple float numbers.
+# float total = float:sum(1.2, 2.3, 3.4);
+#
+# // Take the sum of existing float number list.
+# float[] scores = [11.1, 22.2, 33.3];
+# float totalScore = float:sum(...scores);
+#
+# // Take the sum of multiple float numbers with an existing float number.
+# float value = 21.2;
+# float summation = value.sum(10.5, 21, 32.4);
+#```
 #
 # + xs - float values to sum
-# + return - sum of all of parameter `xs`, +0.0 if parameter `xs` is empty
+# + return - sum of all of parameter `xs`, +0.0 if parameter `xs` is empty, NaN if any member in xs is NaN
```

Comment:
Please don't change the documentation, here and in L67. Documentation changes need to be added in/come from the spec.

Reply:
Removed